bot/admin/server/src/main/kotlin/BotAdminVerticle.kt

```diff
@@ -996,7 +996,7 @@ open class BotAdminVerticle : AdminVerticle() {
             setOf(botUser),
             logger<FaqDefinitionRequest>("Save FAQ")
         ) { context, query: FaqDefinitionRequest ->
-            if (query.utterances.isEmpty() && query.title.isBlank() && query.answer.isBlank()) {
+            if (query.utterances.isEmpty() && query.title.isBlank()) {
```

Comment:
ping @rkuffer

Reply:
Well i don't know what isBlank() does but i guess that checking if a string is given or if a full I18nLabel is valid can't be done the same way. On front side submiting a faq without any i18nLoclizedLabel is forbidden

Reply:
No, this is the old code, the answer was a string, hence the check. 
Now it makes no sense to check in this way, because we've switched to an I18nLabel object, which is mandatory.

Reply:
ok thanks 👍️